semana 12/BankAccount.py

```diff
@@ -0,0 +1,36 @@
+class BankAccount:
+
+    def __init__(self):
+        self.balance = 0
+
+    def send_money_to_account(self, amount):
+        if amount > 0:
+            self.balance += amount
+            print(self.balance)
+        else:
+            print("Negative numbers are not allowed")
+
+    def take_out_money(self, amount):
+        if amount > self.balance or amount < 0:
+            print("Insufficient funds")
+        else:
+            self.balance -= amount
+            print(self.balance)
+
+
+class SavingsAccount(BankAccount):
+    min_balance = 100
+
+    def take_out_money(self, amount):
```

Comment:
You have to validate that the withdrawal doesn't leave the balance below the min_balance